src/pybind/mgr/dashboard/frontend/src/app/shared/components/notification-toast/notification-toast.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cds--toast-notification-container">
+  @for (toast of activeToasts$ | async; track toast.key) {
```

Comment:
```suggestion
  @for (toast of activeToasts$ | async; track $index) {
```
this was a mistake in my last review, as `toast.key` is not a unique ID